### PR TITLE
grammar/style update

### PR DESCRIPTION
**Pull request by gj**
- remove incorrect statement; `post.save` does not "[return] the newly created `Post` object"
- rewrite broken sentences (e.g., line 42)
- fix comma splices
- remove first-person narration

### CODE REVIEW

**Comment by AnnJohn** [comment]
'cc @mendelB @PeterBell

File: README.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 # Create Action
 
-So we have a form that can accept data and then print out the the values to the screen, which is great for learning how forms work in Rails. However, in a real life application we'll most likely want to add the data as a new record in the database. In the `CRUD` lifecycle, this is where the `C` comes in and we `create` a new record.
-
-Before implementing this functionality, let's first open up a Rails console session and create a record manually:
+Let's take a deeper look at how the Rails forms we've been creating actually function. In this lesson, we'll code a `create` action that saves a new `Post` object and then redirects to the newly-created post's `show` page. Before implementing this functionality, let's first open up a Rails console session and create a record manually:
```
Comment (mendelB) [inline README.md]: I think it'd be nice to leave in the bit about the `C` in `CRUD`. Whadya think?

File: README.md [hunk 2]
```diff
@@ -11,17 +9,16 @@ post.description = "Desc goes here..."
...
 ```shell
  (0.1ms)  begin transaction
 SQL (0.3ms)  INSERT INTO "posts" ("title", "description", "created_at", "updated_at") VALUES (?, ?, ?, ?)  [["title", "Title Goes Here"], ["description", "Desc goes here..."], ["created_at", "2015-11-23 22:26:43.799742"], ["updated_at", "2015-11-23 22:26:43.799742"]]
  (1.2ms)  commit transaction
 => true
-
 ```
 
-As you can see, the `save` method generates a SQL script that inserts a new record into the database, passing the `title` and `description` parameters into the statement and then returning the newly created `Post` object. At a high level, this is what the `create` method in our controller will be doing.
+As you can see, the `save` method generates a SQL script that inserts a new record into the database, passing each of the `Post` object's attributes into the statement. At a high level, this is what the `create` method in our `PostsController` will be doing.
```
Comment (mendelB) [inline README.md]: Perhaps we should fill the void of the return the newly created post object statement with '...and then returning `true` indicating that the object was saved successfully' or something of the sort?

**Comment by mendelB** [comment]
@gj Left two small notes. Let me know watcha think

**Comment by gj** [comment]
Done and done, @mendelB

**Pull request merged**